### ISSUE TITLE
rel="feed" is used with Atom feeds

### ISSUE THREAD

**Issue by Minoru** [DanySpin97/danyspin97.github.io#2]
Your HTML templates include `<link rel="feed">` with `href` pointing to an ordinary Atom feed. [According to IndieWeb](https://indieweb.org/rel-feed), `rel="feed"` should be used with h-feeds, which are totally different from Atom. I'm not aware of any newsreaders that actually support h-feeds, but anyway, this might confuse some of them if they expect h-feed but get Atom instead. Please either change the `href` or remove those `<link>`s altogether.

**Comment by ghost** [comment]
What about change it in:

```
<link rel="alternate" type="application/rss+xml" title="Feed" href="http://example.com">
```

**Comment by Minoru** [comment]
@domcorvasce, similar entry already exists, pointing at https://danyspin97.github.io/index.xml. Or am I misunderstanding your suggestion?

**Comment by danyspin97** [comment]
Thank you for pointing this out.
I use [gohugo] to generate the static site and according to [gohugo documentation], it uses both `rel="feed"` and `rel="alternate"`.
The [html5 documentation] suggest the same thing you said, so the best thing to do is pointing this out at [gohugo github project].

However I'm really not into html and rss feed to open a well documented issue.

[gohugo]: https://gohugo.io
[gohugo documentation]: https://gohugo.io/templates/rss/
[gohugo github project]: https://github.com/gohugoio/hugo
[html5 documentation]: http://www.html-5.com/tags/link-tag/index.html#rel-feed

**Comment by ghost** [comment]
@Minoru you're right.
Excuse my ignorance but should be it enough?

**Comment by Minoru** [comment]
@DanySpin97 

> according to gohugo documentation, it uses both rel="feed" and rel="alternate".

Not exactly. These lines don't affect Hugo—it will generate the feed anyway. These lines are about *linking* to that feed, and it's up to *you* how to do that; the docs merely provide an example. I've opened [an issue with Hugo](https://github.com/gohugoio/hugoDocs/issues/369) to correct that example.

html-5.com page seem to be based on an 8-year-old HTML5 draft, which differs significantly from the final HTML 5 specification. If you ever need to learn something about HTML again, check out [the specification itself](https://www.w3.org/TR/html5/)—it's up-to-date by definition, and it's surprisingly readable for something so formal ;)
@domcorvasce 

Sorry, I didn't mean to come across as haughty or snobby. Yes, a single `<link>` with `rel="alternate"` is enough.

**Comment by ghost** [comment]
@Minoru I'm glad you did it, there is always something to learn :smile:

**Comment by danyspin97** [comment]
@Minoru 
First of all, thank you for your detailed explanation.
You're right, the problem was in the theme who used the gohugo documentation example. I've opened a [pull request](https://github.com/shenoybr/hugo-goa/pull/41) to fix this.

I didn't know the HTML 5 specification site but from now on it will be the first site I'll open if I need something about HTML 5.

And it's totally right and not snobby to fix something that's broken.

**Comment by Minoru** [comment]
Cool! I don't know that much about Hugo, but if I understand [this part of the docs](https://gohugo.io/templates/output-formats/#output-formats-for-pages) correctly, you'll have to add a new section to your config once that PR is merged and you update to new version.

**Comment by Minoru** [comment]
…Or maybe you won't have to. I'm a bit confused, and I don't want to read any more docs or install Hugo to find out. If any of you understand Hugo well enough, i encourage you to submit a PR to fix gohugoio/hugoDocs#369.

Thanks for discussion, guys!

**Comment by Minoru** [comment]
Your PR fixing my GoHugo issue was merged, and this issue is fixed by https://github.com/DanySpin97/danyspin97.github.io/commit/c5313e37c34053624acaf50f8fe979f07caef654, so I'm closing it. Thanks for the work! \o/
